### PR TITLE
Create Mesh Export Service Integration Test

### PR DESCRIPTION
**Pull request by andremig-bentley**
WIP

Currently facing webpack issues, creating this PR to get help. Not ready for review.

### CODE REVIEW

File: extensions/frontend-tiles/src/integration-test/utils/webpack-config.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
+// /*---------------------------------------------------------------------------------------------
```
Comment (andremig-bentley) [inline extensions/frontend-tiles/src/integration-test/utils/webpack-config.js:1]: webpack-config is here.

Comment (aruniverse) [reply extensions/frontend-tiles/src/integration-test/utils/webpack-config.js:1]: i think you want to use the [CopyExternalsPlugin](https://github.com/iTwin/itwinjs-core/blob/master/tools/webpack-core/src/plugins/CopyExternalsPlugin.ts#L94)

Comment (andremig-bentley) [reply extensions/frontend-tiles/src/integration-test/utils/webpack-config.js:1]: I've added the plugin, I'm not sure if it's done correctly, I couldn't find examples of it being used in a webpack-config. If it is set up correctly, it doesn't seem to have done much. By removing the call to externalsPreset{node: true}, webpack still doesn't successfully compile/bundle, citing the same errors as before. The NodePolyfillPlugin above it does help to reduce the number of errors signigicantly (250 -> 110), but there are still many present. When the externalsPreset is present, webpack still compiles correctly, but then I cant run the tests and I get the same "require is not defined" error.

File: extensions/frontend-tiles/src/integration-test/utils/webpack-config.js [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+//  * See LICENSE.md in the project root for license terms and full copyright
+//  *notice.
+//  *--------------------------------------------------------------------------------------------*/
+
+const path = require('path');
+const {globSync} = require('glob');
+const frontendLib = path.resolve(__dirname, '../../../lib/cjs');
+const Dotenv = require('dotenv-webpack');
+const webpack = require('webpack');
+const copyExternalsPlugin =
```
Comment (aruniverse) [inline extensions/frontend-tiles/src/integration-test/utils/webpack-config.js:12]: you need to add a dev dep on this and import it from the pkg not this super deep import

Comment (andremig-bentley) [reply extensions/frontend-tiles/src/integration-test/utils/webpack-config.js:12]: I believe that should be fixed now, but same behavior as my previous comment.

**Comment by andremig-bentley** [comment]
Closing this PR as it's no longer necessary. Thank you for the help!

**Pull request closed without merging**